node/tests/GlideClusterClient.test.ts

```diff
@@ -68,9 +69,33 @@ describe("GlideClusterClient", () => {
         cluster = clusterAddresses
             ? await ValkeyCluster.initFromExistingCluster(
                   parseEndpoints(clusterAddresses),
+                  async (addresses: [string, number][]) => {
+                      let glideClient = await GlideClient.createClient(
```

Comment:
cluster client - standalone may (should?) fail

Reply:
Updated.